src/docs/app-monitoring/Sysdig-Monitoring-Guidline-for-shared-services-and-apps.md

```diff
@@ -0,0 +1,276 @@
+---
+title: SRE Guideline for Platform Shared Services
+
+slug: Sysdig-monitor-setup
+
+description: Service Golden Signal - monitoring standards and best practices that will be applied to Platform Shared Services.
+
+keywords: SRE, Sysdig, Sysdig monitor, SLI, monitoring, OpenShift monitoring, developer guide, team guide, team, configure
+
+page_purpose: Documented monitoring metrics and monitoring tools that will describe the approach to monitoring and alerting that will be applied to all platform-shared services and apps.
+
+audience: developer, technical lead
+
+author: Billy Li
+
+content_owner: Billy Li
+
+sort_order: 2
+---
+
+## Intro
+
+SRE, which stands for Site Reliability Engineering, has become an important concept to ensure an application's reliability by quickly returning the system to a steady state. Software can be used as a tool to gain full visibility into the health of an application or system, allowing us to identify and resolve issues before they impact stakeholders.
+
+In this document, we will explore the basic concept of SRE and demonstrate how to set it up using the Registry application as an example.
+
+
+#### Registry
+The registry is an application where teams can submit requests for provisioning namespaces in OpenShift 4 (OCP4) clusters. The registry allows teams to:
```

Comment:
let's call it `BC Platform Services Product Registry` for more accurate naming ;)

Reply:
maybe helpful to describe the project structure: API, web frontend, DB, automation provisioner, etc...